src/interfaces/IValidation.sol

```diff
@@ -27,6 +28,7 @@ interface IValidation is IPlugin {
     /// @param data The calldata sent.
     /// @param authorization Additional data for the validation function to use.
     function validateRuntime(
```

Comment:
Thanks for updating these other two validation functions - with this, we should have full composability across all of the different types of validation functions.